salt/utils/etcd_util.py

```diff
@@ -897,8 +962,8 @@ def get_conn(opts, profile=None, **kwargs):
     if use_v2:
         salt.utils.versions.warn_until(
             "Potassium",
-            "Starting with the Potassium release, etcd API v3 will be the default "
-            "and etcd API v2 will be deprecated.",
+            "etcd API v2 has been deprecated.  It will be removed in "
+            "the Potassium release, and etcd API v3 will be the default.",
```

Comment:
**comment** technically it won't be the default, but the only option, if v2 is removed :trollface: